index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Israel CHIZUNGU Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use the link below instead to help you apply the **font-weight** on text when you use them.
```
<link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:ital,wght@0,500;0,600;1,700&display=swap" rel="stylesheet">
```